generator/geometry_holder.hpp

```diff
@@ -0,0 +1,249 @@
+#pragma once
+
+#include "generator/feature_builder.hpp"
+#include "generator/feature_generator.hpp"
+#include "generator/feature_helpers.hpp"
+#include "generator/tesselator.hpp"
+
+#include "geometry/distance.hpp"
+#include "geometry/point2d.hpp"
+#include "geometry/polygon.hpp"
+#include "geometry/simplification.hpp"
+
+#include "indexer/data_header.hpp"
+#include "indexer/geometry_serialization.hpp"
+
+#include <cstdint>
+#include <functional>
+#include <list>
+#include <vector>
+
+// namespace feature
+//{
+class GeometryHolder
+{
+public:
+  using FileGetter = std::function<FileWriter &(int i)>;
+  using Points = std::vector<m2::PointD>;
+  using Polygons = std::list<Points>;
+
+  // For FeatureType serialization maxTrianglesNumber should be < numeric_limits<uint8_t>::max
+  // because FeatureType format uses uint8_t to encode triangles number.
```

Comment:
optional: s/triangles number/the number of triangles/

Reply:
done